src/app/nation/nation.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { Location } from '@angular/common';
+import { Router } from '@angular/router';
+import { CouchService } from '../shared/couchdb.service';
+
+@Component({
+  selector: 'app-nation',
+  templateUrl: './nation.component.html',
+  styleUrls: ['./nation.component.scss']
+})
+export class NationComponent implements OnInit {
+  message = '';
+  constructor(
+    private couchService: CouchService,
+  	private router: Router
+  ) { }
+
+  ngOnInit() {
+  }
+
+  onSubmit(nation) {
+    if(nation.nation_name !== "" && nation.nationurl !== "" && nation.type !=="") {
+      this.couchService.post('nations', {'adminname':nation.admin_name, 'nationname': nation.nation_name,'nationurl':nation.nationurl, 'type':nation.type}, )
+        .then((data) => {
+        alert('Nation has been sucessfully created');
+        this.router.navigate(['']);
+      }, (error) => this.message = 'Error');
+    }else{
```

Comment:
Please add spaces here to change it to `} else {`.

Before pushing code please make sure it runs through the linter either with the git hook or you can run it manually on your own.